docs/variants/msi_z690/hardware-matrix.md

```diff
@@ -57,7 +57,7 @@ please check the [FAQ section](./faq.md).
 
 | Component                      | Description                                 |
 |--------------------------------|---------------------------------------------|
-| **CPU**                        | Intel Core i5-12600K 3.7                    |
+| **CPU**                        | Intel Core i5-13600K 3.5                    |
```

Comment:
Again what is this 3.5?

Why Noctua is in the CPU section? It seems like some copy-pasta error in the whole file

Reply:
It was like that before my changes, but I fixed it and some other minor things in c8b8d568e72ec09090576989d4b76fca0f49a370